lib/utils/cache.py

```diff
@@ -9,14 +9,14 @@
 import sqlite3
 
 import appdirs
-import diskcache
+import diskcache  # type: ignore[import-untyped]
 
 from lib.utils.settings import global_settings
 
 try:
     from functools import lru_cache
 except ImportError:
-    from backports.functools_lru_cache import lru_cache
+    from backports.functools_lru_cache import lru_cache  # type: ignore[no-redef,import-untyped]
```

Comment:
We can go ahead and get rid of the `try` now, I think.  We're long since using a version of Python that's new enough to have lru_cache.